docs/cli.rst

```diff
@@ -101,6 +101,8 @@ replaces the :meth:`Flask.run` method in most cases. ::
     is provided for convenience, but is not designed to be particularly secure,
     stable, or efficient. See :ref:`deployment` for how to run in production.
 
+To run the development server on a specific interface or port, use the
+``--host`` or ``--port`` arguments.  Run ``flask run --help`` for details.
```

Comment:
Earlier in the doc: "The `--help` option will give more information about any commands and options."

I specifically don't call out `--host` because in most cases, including your added example, people misuse this to make the dev server visible publicly, which is explicitly discouraged. Changing the port is already described later.